drv/lpc55-sprot-server/src/handler.rs

```diff
@@ -26,6 +28,8 @@ task_slot!(DUMPER, dumper);
 
 task_slot!(ATTEST, attest);
 
+task_slot!(SP_CTRL, swd);
```

Comment:
Inertia: everyone else calls it `SP_CTRL`:
```console
$ rg "task_slot.*swd"
task/dumper/src/main.rs
39:task_slot!(SP_CTRL, swd);

drv/lpc55-sprot-server/src/handler.rs
31:task_slot!(SP_CTRL, swd);

task/hiffy/src/lpc55.rs
17:task_slot!(SP_CTRL, swd);

task/sp_measure/src/main.rs
17:task_slot!(SP_CTRL, swd);
```
(maybe because the Idol interface is named `SpCtrl`?)